starter-code/index.html

```diff
@@ -42,11 +43,18 @@ <h1>Pizza Builder</h1>
     <!-- Price -->
     <aside class="panel price">
       <h2>Your pizza's price</h2>
-      <b><!-- Put the base price of the pizza here --> cheese pizza</b>
+      <b>
+        <!-- Put the base price of the pizza here -->
+        Base price: {{basePrice}}$</b>
+
       <ul>
         <!-- List all selected ingredients with their price. Use a computed property. -->
+        <li v-for="ingredient in ingredients" v-if="ingredient.selected"> {{ingredient.name}}: {{ingredient.price}}$
```

Comment:
You could instead create a computed prop that is the array of only selected ingredients, then iterate directly over that array:
`v-for="ingredient in selectedIngredients"`